packages/web-components/src/divider/divider.template.ts

```diff
@@ -0,0 +1,7 @@
+import { ElementViewTemplate, html } from '@microsoft/fast-element';
+import type { Divider } from './divider.js';
+
+/**
+ * @internal
+ */
+export const template: ElementViewTemplate<Divider> = html<Divider>`<slot></slot>`;
```

Comment:
Adding a note here that we'll want to align closely to the react template. A prior attempt I had made while doing some initial examples of Fluent 2 aligned implementations is here: https://github.com/chrisdholt/fluent-v9-example-wc/blob/main/src/divider/divider.template.ts

I think we should try and remove the when directive though if possible...

Reply:
I think replacing the `when` with a ternary operator should work:
`${x => (x.hasChildNodes() ? html`<div class="wrapper" part="wrapper"><slot></slot></div>` : '')}`

Reply:
That'll basically do the same - I just wonder if the cost is even necessary...we can look at it.